Sunday September 1st 2013 11:59 am -04:00,I encounter the same problem! AllPassFailErrorWarningViewstudent4250370DetailSummary: test_get_hours (student)70070student: test_get_hours.TestGetHours.test_example_from_handoutERRORstudent: test_get_hours.TestGetHours.test_large_inputERRORstudent: test_get_hours.TestGetHours.test_seconds_in_afternoonERRORstudent: test_get_hours.TestGetHours.test_seconds_just_after_hourERRORstudent: test_get_hours.TestGetHours.test_seconds_just_before_hourERRORstudent: test_get_hours.TestGetHours.test_seconds_on_hourERRORstudent: test_get_hours.TestGetHours.test_seconds_zeroERRORSummary: test_get_minutes (student)50050student: test_get_minutes.TestGetMinutes.test_example_from_handoutERRORstudent: test_get_minutes.TestGetMinutes.test_just_after_minuteERRORstudent: test_get_minutes.TestGetMinutes.test_just_before_minuteERRORstudent: test_get_minutes.TestGetMinutes.test_seconds_on_minuteERRORstudent: test_get_minutes.TestGetMinutes.test_seconds_zeroERRORSummary: test_get_seconds (student)40040student: test_get_seconds.TestGetSeconds.test_example_from_handoutERRORstudent: test_get_seconds.TestGetSeconds.test_greater_than_60ERRORstudent: test_get_seconds.TestGetSeconds.test_less_than_60ERRORstudent: test_get_seconds.TestGetSeconds.test_seconds_zeroERRORSummary: test_hours_difference (student)70070student: test_hours_difference.TestHoursDifference.test_examples_from_docstringERRORstudent: test_hours_difference.TestHoursDifference.test_time_1_much_larger_than_time_2ERRORstudent: test_hours_difference.TestHoursDifference.test_time_1_much_smaller_than_time_2ERRORstudent: test_hours_difference.TestHoursDifference.test_time_1_same_as_time_2ERRORstudent: test_hours_difference.TestHoursDifference.test_time_1_slightly_larger_than_time_2ERRORstudent: test_hours_difference.TestHoursDifference.test_time_1_slightly_smaller_than_time_2ERRORstudent: test_hours_difference.TestHoursDifference.test_times_both_zeroERRORSummary: test_seconds_difference (student)55000student: test_seconds_difference.TestSecondsDifference.test_examples_from_docstringSUCCESSstudent: test_seconds_difference.TestSecondsDifference.test_time_1_larger_than_time_2SUCCESSstudent: test_seconds_difference.TestSecondsDifference.test_time_1_same_as_time_2SUCCESSstudent: test_seconds_difference.TestSecondsDifference.test_time_1_smaller_than_time_2SUCCESSstudent: test_seconds_difference.TestSecondsDifference.test_times_both_zeroSUCCESSSummary: test_time_from_utc (student)40040student: test_time_from_utc.TestTimeFromUTC.test_day_aheadERRORstudent: test_time_from_utc.TestTimeFromUTC.test_day_behindERRORstudent: test_time_from_utc.TestTimeFromUTC.test_examples_from_docstringERRORstudent: test_time_from_utc.TestTimeFromUTC.test_minutes_nonzeroERRORSummary: test_time_to_utc (student)40040student: test_time_to_utc.TestTimeToUTC.test_day_aheadERRORstudent: test_time_to_utc.TestTimeToUTC.test_day_behindERRORstudent: test_time_to_utc.TestTimeToUTC.test_examples_from_docstringERRORstudent: test_time_to_utc.TestTimeToUTC.test_minutes_nonzeroERRORSummary: test_to_float_hours (student)60060student: test_to_float_hours.TestToFloatHours.test_examples_from_docstringERRORstudent: test_to_float_hours.TestToFloatHours.test_hours_minutes_seconds_all_zeroERRORstudent: test_to_float_hours.TestToFloatHours.test_hours_minutes_seconds_nonzeroERRORstudent: test_to_float_hours.TestToFloatHours.test_hours_nonzero_minutes_seconds_zeroERRORstudent: test_to_float_hours.TestToFloatHours.test_minutes_nonzero_hours_seconds_zeroERRORstudent: test_to_float_hours.TestToFloatHours.test_seconds_nonzero_hours_minutes_zeroERRORTotal4250370, Resolved Locked,0
Sunday September 15th 2013 10:42 am -04:00,Can someone please explain to me why i am not geting the full 24/24 score?  The first failure is bellow:      student: test_a2.TestGetComplementarySequence.test_get_complementary_sequence_handout_exampleFAILURE**Method Description:**  Have you tested your function on the example in the handout?  **Message:**  FAILURE: AssertionError     != TA    + TA : Have you tested your function on the example in the handout?The first failure is :student: test_a2.TestGetComplementarySequence.test_get_complementary_sequence_longFAILURE[x]Method Description:Have you tested your function on a sequence with multiple occurrences        of each nucleotide?Message:FAILURE: AssertionError != TAGCTAGC+ TAGCTAGC : Have you tested your function on a sequence with multiple occurrences            of each nucleotide?and my last failure is :student: test_a2.TestGetComplement.test_get_complementFAILURE[x]Method Description:Have you tested your function with all 4 nucleotides?Message:FAILURE: AssertionErrorNone != T : Have you tested your function with all 4 nucleotides?WHAT DO YOU SUPPOSE I AM DOING WRONG. I TESTED EVERYTHING IN IDLE AND ALL WORKING GREAT...IS IT AN AUTOGRADE PROBLEM?, Unresolved,1
Sunday September 15th 2013 2:25 pm -04:00,no i have no print at all, Unresolved,0
Sunday September 15th 2013 2:30 pm -04:00,these are my examples and all work pretty much fine.def get_complement    >>> get_complement (A)    T    >>> get_complement (T)    A    >>> get_complement (C)    G    >>> get_complement (G)    Cdef get_complementary_sequence    >>> get_complementary_sequence (ATCG)    TAGC    >>> get_complementary_sequence (CG)    GC    >>> get_complementary_sequence (CA)    GT    >>> get_complementary_sequence (TGATC)    ACTAG, Unresolved,0
Sunday September 15th 2013 2:35 pm -04:00,well i just resubmited my codes and got 24/24 whithout touching anything...Houston autograde has a problem!!!, Unresolved,0
